expenses/tests_expenses.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from django.utils import timezone
+from .models import Expenses
+
+AMOUNT = '100'
+DATE = timezone.now()
+CATEGORY = 'CLOTHING'
+
+
+@pytest.fixture
+def expense_test0():
+    return Expenses(amount=AMOUNT,
+                    date=DATE,
+                    Category=CATEGORY
+                    )
+
+
+@pytest.mark.django_db
+class TestExpensesModel:
+    def test_create_expense(self, expense_test0):
+        assert expense_test0.amount == AMOUNT
+        assert expense_test0.date == DATE
+        assert expense_test0.Category == CATEGORY
+
+
+def test_save_expense(self, expense_test0):
+    expense_test0.save()
+    assert expense_test0 in Expenses.objects.all()
+
+
+def test_delete_expense(self, expense_test0):
+    expense_test0.save()
+    expense_test0.delete()
+    assert expense_test0 not in Expenses.objects.all()
```

Comment:
are these suppose to be in the class TestExpenseModel?(self in params)
if so they are missing an indentation

Reply:
also I like the idea of testing the save and delete separately ill implement that in the House app as well, good one 🙌

Reply:
fixed

> are these suppose to be in the class TestExpenseModel?(self in params) if so they are missing an indentation